ui/main.qml

```diff
@@ -214,11 +214,11 @@ StatusWindow {
             } else {
                 if (loader.sourceComponent != app) {
                     Qt.quit();
-                }
-                else if (loader.sourceComponent == app) {
+                } else if (loader.sourceComponent == app) {
                     if (localAccountSensitiveSettings.quitOnClose) {
                         Qt.quit();
                     } else {
+                        close.accepted = false
```

Comment:
`close.accepted = true` seems to be not necessary as `true` is the default value.

Reply:
But I have a `false` here as we want to prevent the closing :)